docker-compose.yml

```diff
@@ -8,6 +8,16 @@ services:
       - .:/usr/chapter
     ports:
       - '8000:8000'
```

Comment:
we should also ensure that `app` depends on `db` using docker-compose's `depends_on` (otherwise server may try to connect before postgres initializes)